README.md

```diff
@@ -107,7 +107,19 @@ See Installation instructions for:
     # remember to source the setup.bash
     source install/share/setup.bash
 
-    ros2 launch mbzirc_ign spawn.launch.py name:=usv world:=simple_demo model:=wam-v type:=usv x:=20 y:=0 z:=-1.0 R:=0 P:=0 Y:=0
+    ros2 launch mbzirc_ign spawn.launch.py name:=usv world:=simple_demo model:=wam-v type:=usv x:=0 y:=0 z:=-1.0 R:=3.14 P:=0 Y:=0
```

Comment:
the USV is spawned inside the platform with z=-1.0 for me. 1.5 seems to work and puts it on top of the platform

Reply:
I forgot up update `z`. 5ac0e4a